draft-nygren-dnsop-svcb-httpssvc.md

```diff
@@ -627,6 +627,28 @@ in cache before performing any followup queries.
 With these optimizations in place, and conforming DNS servers,
 using SVCB does not add network latency to connection setup.
 
+Moreover, if an A or AAAA response arrives before an SVCB response, the
+client SHOULD wait up to CD milliseconds before connecting as if the
+SVCB query returned NODATA, but MUST NOT transmit any information that
+could be altered by an SVCB response until the SVCB response arrives.
+For example, a TLS connection may be altered by the "esnikeys" value
+of an SVCB response. CD (Connection Delay) is a configurable parameter.
+The recommended value is 50 milliseconds, as per the guidance in {{!HappyEyeballsV2=RFC8305}}.
+
+If an SVCB record is consistent with an active or in-progress connection,
+the client may continue using that connection using any information provided
+by the SVCB RR. For example, if an SVCB record with a "esnikeys" value and
+"ipv4hint" or "ipv6hint" values arrives and one of those hints matches the
```

Comment:
Maybe it would be clearer to say
> if an SVCB record arrives with an "esnikeys" value and an IP address from "ipv6hint" or an attached AAAA record that matches an in-progress connection, ...

That doesn't cover every case it but clarifies what is meant by "consistent".

Reply:
I think this is OBE.